package.json

```diff
@@ -180,7 +180,7 @@
   "dependencies": {
     "@hapi/call": "^8.0.0",
     "@hapi/subtext": "^7.0.3",
-    "@kubernetes/client-node": "^0.12.0",
+    "@kubernetes/client-node": "^0.14.3",
```

Comment:
Why this is needed?

Reply:
I upgraded because it was available and I was trying out to see if it naively supported the getting of all resource lists (including legacy APIs).

It didn't but since we now also support 1.21 for kube, it seems like we should keep the upgrade. But I can understand if it should be its own PR.